lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -317,6 +321,15 @@ static bool isAcceleratorOnly(const SILFunction &hostFn) {
          SILFunctionType::Representation::TensorFlow; // @convention(tensorflow)
 }
 
+/// Return true if a user instruction is returning or forming a return value.
+static bool isReturning(SILInstruction *user) {
```

Comment:
can we move this definition below, to be right before its use?

Reply:
Done